api/external/applications/applications.proto

```diff
@@ -274,7 +278,6 @@ service ApplicationsService {
 
   Changes the configuration for the task that marks services as disconnected after
   'threshold'. Threshold is a string that follows Elasticsearch's date math expressions.
-  This job cannot be disabled, and therefore no information about running is accepted.
```

Comment:
The example requests should show all the options. Also need some wordage to encourage users to not actually configure the recurrence or disable.